Dockerfile

```diff
@@ -145,21 +146,20 @@ ENV MKL_NUM_THREADS=1 \
 RUN python -c "from matplotlib import font_manager" && \
     sed -i 's/\(backend *: \).*$/\1Agg/g' $( python -c "import matplotlib; print(matplotlib.matplotlib_fname())" )
 
+# Installing dev requirements (packages that are not in pypi)
+WORKDIR /src/
+COPY requirements.txt requirements.txt
+RUN pip install --no-cache-dir -r requirements.txt && \
+    rm -rf ~/.cache/pip
+
 # Precaching atlases
-RUN pip install --no-cache-dir "templateflow<0.4.0a0,>=0.3.0" && \
-    python -c "from templateflow import api as tfapi; \
+RUN python -c "from templateflow import api as tfapi; \
                tfapi.get('MNI152Lin|MNI152NLin2009cAsym|OASIS30ANTs', suffix='T1w'); \
```

Comment:
do the regexes work in this context?

Reply:
No, but it kind of doesn't matter because in these contexts it fetches the needed files. The problem in smriprep was that however the files were referenced, it didn't trigger download.

But I'll update these ones, too.